[release-2.4.0 < T0976-MG] Add command to return startup config (#495)
memgraph/docs: docs/reference-guide/configuration.md

Each configuration setting is in the form: `--setting-name=value` .

> Privilege level `CONFIG` is required to be able to fetch the configuration and it can be attained by calling the following query:

Reviewer: ```suggestion
Privilege level `CONFIG` is required to be able to fetch the configuration via the following query:
```
